app/pods/publications/publication/controller.js

```diff
@@ -79,10 +81,19 @@ export default class PublicationController extends Controller {
 
   @restartableTask
   *setPublicationNumber(event) {
```

Comment:
Je aanpasssingen lijken geen rekening te houden met de deboucing die er op de input zat. Elke letter die je in dit vak typt, gaat onmiddelijk een network request sturen om te checken of het nummer al bestond ... Best de network request achter de timeout steken denk ik.